specification/toolingservice/data-plane/1.0/toolingservice.json

```diff
@@ -0,0 +1,755 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Tooling Service Client",
+    "description": "Client that can be used to access Tooling services to run linter, static and api validations.",
+    "version": "1.0"
+  },
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "host": "<TBD>",
+  "basePath": "/api/",
+  "schemes": [
+    "https", "http"
+  ],
+  "paths": {
+    "/tools/lint": {
+      "post": {
+        "tags": [
+          "Linter"
+        ],
+        "operationId": "Tools_Linter",
+        "description": "Get Linter Result",
+        "parameters": [
+          {
+            "name":"repository",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the repository"
+          },
+          {
+            "name":"branch",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Name of the specific branch in the repository."
+          },
+          {
+            "name":"configPath",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "Configuration path of the file."
+          },
+          {
+            "name":"apiTag",
+            "in": "body",
+            "type": "string",
+            "required": true,
+            "description": "API Tag to be validated."
+          },
+          {
+            "name":"commitHash",
+            "in": "body",
+            "type": "string",
+            "description": "Commit hash to be validated."
+          },
+          {
+            "name":"mode",
+            "in": "query",
+            "type": "string",
+            "description": "Mode"
```

Comment:
Can we be more descriptive? what's mode?

Reply:
Done